tutorials/tensordictmodule.ipynb

```diff
@@ -0,0 +1,931 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3be0fafd",
+   "metadata": {},
+   "source": [
+    "# TensorDictModule"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "94bd315a",
+   "metadata": {},
+   "source": [
+    "We recommand reading the TensorDict tutorial before going through this one."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0652352c",
+   "metadata": {},
+   "source": [
+    "For a convenient usage of the `TensorDict` class with `nn.Module`, TorchRL provides an interface between the two named `TensorDictModule`. <br/>\n",
+    "The `TensorDictModule` class is an `nn.Module` that takes a `TensorDict` as input when called. <br/>\n",
+    "It is up to the user to define the keys to be read as input and output."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5b0241ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
```

Comment:
it feels weird to have this there. In a tutorial, every bit of code should relate to what's above